### PR TITLE
Changes to support mac/linux

### PR DESCRIPTION
**Pull request by christopheranderson**
Changes:
- Added generateProtos.sh (mostly a copy of the batch script)
- Added a new Build target for Grpc project to support mac/linux
- Changed how process info start starts the application (and how Node.js described itself)

### CODE REVIEW

File: src/WebJobs.Script.Grpc/WebJobs.Script.Grpc.csproj [hunk 1]
```diff
@@ -12,10 +12,17 @@
     </PackageReference>
   </ItemGroup>
 
-  <Target Name="GenerateProtoFiles" BeforeTargets="BeforeBuild">
+  <Target Condition="'$(OS)' == 'Windows_NT'" Name="GenerateProtoFiles" BeforeTargets="BeforeBuild">
```
Comment (fabiocav) [inline src/WebJobs.Script.Grpc/WebJobs.Script.Grpc.csproj]: Refactor this to introduce a command name property that is properly set based on the OS, that way we don't need to duplicate the target

**Comment by fabiocav** [comment]
Small comment. Can we also have a more descriptive commit message?

**Comment by mathewc** [comment]
Most of these files appear to be from a different WIP PR (Matt's PR)? What is the actual proposed change in this PR?

**Comment by mamaso** [comment]
@mathewc I messed up the history with a rebase, the important stuff is 1521b2f which I'll pull into my branch

**Pull request closed without merging**